lib/naughty/layout/legacy.lua

```diff
@@ -211,6 +211,7 @@ local escape_subs    = { ['<'] = "&lt;", ['>'] = "&gt;", ['&'] = "&amp;" }
 
 -- Cache the markup
 local function set_escaped_text(self)
+    -- Only handle notifications using the legacy layout.
```

Comment:
A quick look at the last commit: This commit message and the patch do not fit together. The commit message talks about "if the notification has no widget" while the patch only adds a comment.

Could you re-read your commits and check for things that went "weird" in rebases?

Reply:
I updated the commit messages and dropped that "failed rebase" commit you mentioned. There is some issues with some commits (like typo fixed in the wrong one, etc), but hopefully nothing that will be an issue when using git blame or bisecting. So I am merging this now given there is no other reviews and I want to move on. This has been in the merge queue since summer 2017. Some commits are celebrating their second anniversary in the coming weeks...